fuzzing/private/fuzz_test.bzl

```diff
@@ -91,15 +85,20 @@ def cc_fuzz_test(
         )
 
     fuzzing_launcher(
-        name = name + "_run",
-        binary = name,
-        # Since the script depends on the _fuzz_test above, which is a cc_test,
-        # this attribute must be set.
-        testonly = True,
+        name = launcher_name,
+        binary = instrum_binary_name,
+    )
+
+    fuzzing_regression_test(
+        name = name,
+        binary = instrum_binary_name,
+        tags = (tags or []) + [
+            "fuzz-test",
+        ],
     )
 
     oss_fuzz_package(
         name = name + "_oss_fuzz",
```

Comment:
this may be unrelated - should we document this rule?

Reply:
Great catch, done!